### PR TITLE
BV: add entries for  Salt migration minion

### PR DESCRIPTION
**Pull request by NamelessOne91**
Related to https://github.com/SUSE/spacewalk/issues/22410

`salt-migration-minion` and the likes are just placeholder names

### CODE REVIEW

File: terracumber_config/tf_files/SUSEManager-4.3-build-validation-NUE.tf [hunk 1]
```diff
@@ -804,6 +804,25 @@ module "sles15sp5s390-minion" {
...
 }
 
+module "salt-migration-minion" {
+  source = "./modules/minion"
+  base_configuration = module.base.configuration
+  name               = "min-salt-migration"
+  product_version    = "4.3-released"
+  image              = "sles15sp5o"
+  provider_settings = {
+    mac                = "aa:b2:92:42:00:cf"
```
Comment (Bischoff) [inline terracumber_config/tf_files/SUSEManager-4.3-build-validation-NUE.tf:814]: are you sure?
```
infrastructure/srv/salt/dhcpd-server> grep -Ri aa:b2:92:42:00:cf
infrastructure/srv/salt/dhcpd-server> 
```

File: terracumber_config/tf_files/SUSEManager-4.3-build-validation-PRV.tf [hunk 2]
```diff
@@ -1031,6 +1031,25 @@ module "sles15sp5s390-minion" {
...
 }
 
+module "salt-migration-minion" {
+  source = "./modules/minion"
+  base_configuration = module.base.configuration
+  name               = "min-salt-migration"
+  product_version    = "4.3-released"
+  image              = "sles15sp5o"
+  provider_settings = {
+    mac                = "aa:b2:92:42:00:cf"
```
Comment (Bischoff) [inline terracumber_config/tf_files/SUSEManager-4.3-build-validation-PRV.tf:1041]: are you sure?
```
infrastructure/srv/salt/dhcpd-server> grep -Ri aa:b2:92:42:00:cf
infrastructure/srv/salt/dhcpd-server> 
```

File: terracumber_config/tf_files/Uyuni-Master-build-validation-NUE.tf [hunk 3]
```diff
@@ -637,6 +637,25 @@ module "sles15sp5s390-minion" {
...
 }
 
+module "salt-migration-minion" {
+  source = "./modules/minion"
+  base_configuration = module.base.configuration
+  name               = "min-salt-migration"
+  product_version    = "uyuni-master"
+  image              = "sles15sp5o"
+  provider_settings = {
+    mac                = "aa:b2:93:02:01:cf"
```
Comment (Bischoff) [inline terracumber_config/tf_files/Uyuni-Master-build-validation-NUE.tf:647]: Are you sure?
```
galaxy/infrastructure/srv/salt/dhcpd-server> grep -Ri aa:b2:93:02:01:cf
galaxy/infrastructure/srv/salt/dhcpd-server> 
```

File: terracumber_config/tf_files/Uyuni-Master-build-validation-PRV.tf [hunk 4]
```diff
@@ -839,6 +839,26 @@ module "sles15sp5s390-minion" {
...
 }
 
+module "salt-migration-minion" {
+  source = "./modules/minion"
+  base_configuration = module.base.configuration
+  name               = "min-salt-migration"
+  product_version    = "uyuni-master"
+  image              = "sles15sp5o"
+  provider_settings = {
+    mac                = "aa:b2:93:02:01:9b"
```
Comment (Bischoff) [inline terracumber_config/tf_files/Uyuni-Master-build-validation-PRV.tf:849]: Are you sure?
```
galaxy/infrastructure/srv/salt/dhcpd-server> grep -Ri aa:b2:93:02:01:9b
galaxy/infrastructure/srv/salt/dhcpd-server> 
```

**Review by Bischoff — CHANGES_REQUESTED** [review]

**Review by srbarrios — COMMENTED** [review]
Once MACs are fixed, the rest looks good to me.

File: terracumber_config/tf_files/SUSEManager-4.3-build-validation-NUE.tf (hunk 1 above)
Comment (NamelessOne91) [reply terracumber_config/tf_files/SUSEManager-4.3-build-validation-NUE.tf:814]:
```
infrastructure/srv/salt> grep -Ri aa:b2:92:42:00:cf
dhcpd-server/mgr_prv_suse_net/dhcpd.conf:host suma-bv-43-min-salt-migration { hardware ethernet aa:b2:92:42:00:cf; fixed-address 10.84.221.207; option host-name "suma-bv-43-min-salt-migration"; }
dhcpd-server/mgr_suse_de/dhcpd.conf:host suma-bv-43-min-salt-migration   { hardware ethernet aa:b2:92:42:00:cf; fixed-address 10.145.209.207; option host-name "suma-bv-43-min-salt-migration";  }
```

I've got them. Maybe you did not pull the last changes ? ( same for the other comments)

Comment (Bischoff) [reply terracumber_config/tf_files/SUSEManager-4.3-build-validation-NUE.tf:814]: Right, i forgot the git pull

**Review by Bischoff — APPROVED** [review]

**Pull request merged**